tests/group/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required (VERSION 3.0) # The minimum version of CMake necessary to build this project
+project (group) # The name of our project
+
+declare_trisycl_test(TARGET group TEST_REGEX
+" 10
+ 2
+ 0
+ 5
+5 
```

Comment:
A lot of trailing spaces starting from here. Are they part of the regex and meaningful?